docs/pages/kubernetes-access/guides/discovery/aws.mdx

```diff
@@ -0,0 +1,215 @@
+performing discovery requires an IAM policy that allows Teleport to list and 
+describe EKS clusters:
+
+```js
+{
+    "Version": "2012-10-17",
+    "Statement": [
+        {
+            "Effect": "Allow",
+            "Action": [
+                "eks:DescribeCluster",
+                "eks:ListClusters"
+            ],
+            "Resource": "*"
+        }
+    ]
+}      
+```
+
+## Step 2/3. Configure EKS authentication
+
+
+When `kubernetes_service` uses an IAM role that is different from the one that 
+created the cluster, it is necessary to edit the `configmap/aws-auth` 
+and add the mapping between the IAM role that the Teleport process is using and the 
+RBAC permissions for each of the desired clusters. You can take advantage of EKS 
+cluster creation and management tools such as `eksctl` and Terraform to automatically 
+configure access.
+
+For simplicity, it is possible to set up an existing group in the cluster to the 
+Teleport IAM role such as `system:masters`, but recommend setting up the minimum 
+permissions as described below.
+
+<Details 
+title="Configure Teleport minimal RBAC permissions" 
+opened={false}
+>
+
+Connect to your target cluster with admin permissions and create the following 
+resources using `kubectl`.
+
+### ClusterRole
+
+Create the `ClusterRole` RBAC definition with minimum permissions for the Teleport 
+to forward requests to the cluster.
+
+```yaml
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: teleport
+rules:
+- apiGroups:
+  - ""
+  resources:
+  - users
+  - groups
+  - serviceaccounts
+  verbs:
+  - impersonate
+- apiGroups:
+  - ""
+  resources:
+  - pods
+  verbs:
+  - get
+- apiGroups:
+  - "authorization.k8s.io"
+  resources:
+  - selfsubjectaccessreviews
+  - selfsubjectrulesreviews
+  verbs:
+  - create
+```
+
+### ClusterRoleBinding
+
+Link the previously created `ClusterRole` into an RBAC group.
+
+```yaml
+kind: ClusterRoleBinding
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: teleport
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: teleport
+subjects:
+- kind: Group
+  name: teleport
+  apiGroup: rbac.authorization.k8s.io
+```
+
+Finally, edit the `configmap/aws-auth` in the `kube-system` namespace to append 
+the following `mapRoles`. Please replace `{teleport_aws_iam_role}` with the 
+appropriate IAM role. This step will link the Teleport IAM role into the Kubernetes 
+RBAC group `teleport`.
+
+```yaml
+apiVersion: v1
+data:
+  mapRoles: |
+    - groups:
+      - teleport
+      rolearn: {teleport_aws_iam_role} # e.g. arn:aws:iam::222222222222:role/teleport-role
+      username: teleport
+```
+
+At this point, the Teleport IAM role already has the minimum permissions 
+to forward requests to the cluster. 
+
+</Details>
+
+
+## Step 3/3. Configure Teleport to discover EKS clusters
+
+The Teleport Kubernetes Discovery requires a valid auth token for the `discovery` and `kube` services 
```

Comment:
```suggestion
Teleport EKS Auto-Discovery requires a valid auth token for the `discovery` and `kube` services 
```
To be consistent with other parts of the guide (though are we capitalizing "Auto-Discovery" everywhere?).